tests/zenko_e2e/backbeat/BackbeatAPIUtility.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const { makeGETRequest, makePOSTRequest, getResponseBody } =
+    require('./utils/request');
+
+class BackbeatAPIUtility {
+    constructor() {}
+
+    getReplicationStatus(locationName, cb) {
+        let path = '/_/backbeat/api/crr/status';
+        if (locationName) {
+            path = `${path}/${locationName}`;
+        }
+        makeGETRequest(path, (err, res) => {
+            assert.ifError(err);
+            getResponseBody(res, (err, body) => {
+                assert.ifError(err);
+                return cb(null, body);
+            });
+        });
+    }
+
+    getReplicationResumeSchedule(locationName, cb) {
+        const path = `/_/backbeat/api/crr/resume/${locationName}`;
+        makeGETRequest(path, (err, res) => {
+            assert.ifError(err);
+            getResponseBody(res, (err, body) => {
+                assert.ifError(err);
+                return cb(null, body);
+            });
+        });
+    }
+
+    pauseReplication(locationName, cb) {
```

Comment:
Maybe we want to add a test case which uses the path `${path}/${locationName}`. I don't see a test case in which `pauseReplication` is being called with a particular location name.

Reply:
I was thinking I could do that part when I setup 1-multiple  CRR testing :')